src/utils/metakg/export.py

```diff
@@ -52,29 +55,35 @@ def edges2graphml(chunk, api_call, protocol, host, edge_default="directed"):
     # iterate over edges and fill in data
     for data in edges:
         node = ET.SubElement(graph, 'node')
+        node_id = data['subject']  # Get the node ID
         node.set('id', data['subject'])
-        
+
         data_node = ET.SubElement(node, 'data')
         data_node.set('key', 'd1')
         data_node.text = data['subject']
-        
+
+        unique_node_ids.add(node_id)
+
         edge = ET.SubElement(graph, 'edge')
         edge.set('source', data['subject'])
         edge.set('target', data['object'])
-        
+
         data_edge = ET.SubElement(edge, 'data')
         data_edge.set('key', 'd2')
         data_edge.text = data['predicate']
 
+    # Calculate the node count
+    node_count = len(unique_node_ids)
+
     # tree = ET.ElementTree(root)
-    graphml_string = ET.tostring(root, encoding="utf-8", method="xml").decode()  
-    title_text=f'''This GraphML export was generated from this SmartAPI MetaKG (Meta KnowledgeGraph) query:'''
+    graphml_string = ET.tostring(root, encoding="utf-8", method="xml").decode()
+    title_text='''This GraphML export was generated from this SmartAPI MetaKG (Meta KnowledgeGraph) query:'''
```

Comment:
@NikkiBytes don't use `'''` where you can just use `"`, unless there is a reason I might have missed.

Reply:
@newgene it's updated now and fixed with `fourmat`. Originally the triple quotes were used for a single multi-line, so no need for them now, they have been taken out. 

These edits were tested and I checked that the API endpoint is still working.